doc/whats_new/v0.20.rst

```diff
@@ -444,6 +444,10 @@ Metrics
   :issue:`9515` by :user:`Alan Liddell <aliddell>` and
   :user:`Manh Dao <manhdao>`.
 
+- Fixed a bug in:  :func:`metrics.brier_score_loss`, when all ``y_true`` values are 1,
```

Comment:
drop the first :. Change "1," to "1." and start a new sentence.

Reply:
will do